api/PositionOptions.json

```diff
@@ -116,13 +63,61 @@
           "deprecated": false
         }
       },
+      "secure_context_required": {
+        "__compat": {
+          "description": "Secure context required",
+          "support": {
+            "chrome": {
+              "version_added": "47"
+            },
+            "chrome_android": {
+              "version_added": "47"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
```

Comment:
Same as Edge desktop before it. This seems contrary to the prior data

Reply:
ditto